streaming-architecture.md

```diff
@@ -1,3 +1,9 @@
 ## Better JSON through streams
 
-One of the best features of Node is UNIX-like data streams, but knowing how to integrate them in your web application can be daunting! Node v4 and a couple utility libraries make working with streams in the front and back end much more manageable. Using streams it's possible to write much more efficient applications that scale to handling large amounts of data effortlessly. This talk will introduce developers to UNIX streams, how to use them with Node.js scripts, and how to write web application that use streams as their main means of communication.
+One of the best features of Node is UNIX-like data streams, but knowing how to integrate leverage them in your web application can be daunting! This talk will explore three frontiers of streaming: 
+  
+  - Streaming from module to module in the back end.
```

Comment:
Bullet points are tough... some reviewers like them, but many do not. I'd say it's safer to _not_ use them.